docs/performance/loadtest.md

```diff
@@ -0,0 +1,102 @@
+# Load Testing
+
+This document describes how to perform the load testing in Vald cluster or Vald agent using Vald Load Testing Tools.
+
+## Overview
+
+We develop our own Load Testing Tools to perform the load testing in Vald cluster or Vald agent. You can perform the load test on Vald to test the insert and search performance in Vald.
+
+## Prerequisite
+
+- Vald Cluster or Vald Agent
+
+    Please refer to [here](https://vald.vdaas.org/docs/tutorial/get-started/) for the installation guide of Vald Cluster or Vald Agent.
+
+- Docker
+
+## Install load test tools
+
+We provide a docker image for the load testing. Please download the image using the below command.
+
+```bash
+docker pull vdaas/vald-loadtest
+```
+
+## Configure load test tools
+
+Please refer to the [Sample configuration file](https://github.com/vdaas/vald/blob/master/cmd/tools/cli/loadtest/sample.yaml) to configure the Vald Load Testing Tools.
+
+Here is the important configurations and following the explanation.
+
+| Name        | Description                                                                                                                                                      | Example                                             |
```

Comment:
the table will make us to understand clearly in Github, but it may not be shown all of the below table in the public document framework

Reply:
I think it works maybe.
https://raw.githubusercontent.com/vdaas/vald/master/CHANGELOG.md

Reply:
I meant whether work or not.
I wonder whether the whole of table will be shown without scrollbar or not.
e.g. https://vald.vdaas.org/docs/contributing/coding-style/#table-driven-test